controllers/JobController.js

```diff
@@ -132,101 +111,56 @@ exports.JobsByEmail = [
+  body("skills", "Skills must not be less than 3 character.").notEmpty(),
+  body("locations", "Locations must not be empty.").notEmpty(),
+  body("vacancies", "Vacancies must not be empty.").notEmpty(),
+  (req, res) => {
+    try {
+      const user = req.user;
 
-/**
- * Job update.
- *
- * @param {string}      jobId
- * @param {string}      applicantId
- *
- * @returns {Object}
- */
-exports.JobApplied = [
-	auth,
-	body("jobId", "Job ID must not be empty.").isLength({ min: 1 }).trim(),
-	body("applicantId", "Applicant ID must not be empty.").isLength({ min: 1 }).trim(),
-	sanitizeBody("*").escape(),
-	(req, res) => {
-		try {
-			const errors = validationResult(req);
-			// var Job = new JobModel();
+      if (user.userType !== constants.userTypes.RECRUITER)
+        return apiResponse.unauthorizedResponse(res, "Invalid Permission.");
+
+      const errors = validationResult(req);
+      if (!errors.isEmpty()) {
+        return apiResponse.validationErrorWithData(
+          res,
+          "Validation Error.",
+          errors.array()
+        );
+      } else {
+        var Job = new JobModel({
+          title: req.body.title,
+          description: req.body.description,
+          skills: req.body.skills,
+          locations: req.body.locations,
+          vacancies: req.body.vacancies,
+          postedBy: user.id,
+        });
 
-			if (!errors.isEmpty()) {
-				return apiResponse.validationErrorWithData(res, "Validation Error.", errors.array());
-			}
-			else {
-				if (!mongoose.Types.ObjectId.isValid(req.body.jobId)) {
-					return apiResponse.validationErrorWithData(res, "Invalid Error.", "Invalid Job ID");
-				} else if (!mongoose.Types.ObjectId.isValid(req.body.applicantId)) {
-					return apiResponse.validationErrorWithData(res, "Invalid Error.", "Invalid Applicant ID");
-				} else {
-					JobModel.findById(req.body.jobId, function (err, foundJob) {
-						if (foundJob === null) {
-							return apiResponse.notFoundResponse(res, "Job not exists with this id");
-						} else {
-							//Check authorized user
-							if (foundJob.applicants.includes(req.body.applicantId)) {
-								return apiResponse.unauthorizedResponse(res, "You have already applied for this Job.");
-							} else {
-								//update Job.
-								delete foundJob._id;
-								foundJob.applicants.push(req.body.applicantId);
-								JobModel.findByIdAndUpdate(req.body.jobId, foundJob, {}, function (err) {
-									if (err) {
-										return apiResponse.ErrorResponse(res, err);
-									} else {
-										let JobData = new jobData(foundJob);
-										return apiResponse.successResponseWithData(res, "Job update Success.", JobData);
-									}
-								});
-							}
-						}
-					});
-				}
-			}
-		} catch (err) {
-			//throw error in json response with status 500.
-			return apiResponse.ErrorResponse(res, err);
-		}
-	}
-];
\ No newline at end of file
+        Job.save((err, job) => {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 tabs but found 8 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
				Job.save((err, job) => {
```